components/layouts/PageFooter/FooterIconsSection.vue

```diff
@@ -0,0 +1,50 @@
+<template>
```

Comment:
We could try to factor out the common markup and styles for these sections. What about creating three components: `FooterSection`, `FooterLinksSection` and `FooterIconsSection`.

`FooterSection` would use a slot to accept any content where now is the `AppLink` element.

`FooterLinksSection` would use that and add the links as children and `FooterIconsSeciton` would do the same but with icons, in the same way you're already doing here.

Reply:
I unified both components in one called `FooterSection` that decided what to show based on a variable called `showIconLinks`